backend/test/customerTests/service.test.ts

```diff
@@ -0,0 +1,75 @@
+import * as sinon from 'sinon';
+import sinonStubPromise from 'sinon-stub-promise';
+import { expect } from 'chai';
+import 'mocha';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import { config } from '../../config';
+
+import CustomerService from '../../src/Customer/customer.service';
+import CustomerModel from '../../src/Customer/customer.models';
+import emailService from '../../src/Email/email.service';
```

Comment:
you can remove this import, it is not being used

Reply:
Will do